changelogs/current.yaml

```diff
@@ -34,6 +34,9 @@ bug_fixes:
     :ref:`overprovisioning_factor<envoy_v3_api_field_config.endpoint.v3.ClusterLoadAssignment.Policy.overprovisioning_factor>` and
     :ref:`weighted_priority_health <envoy_v3_api_field_config.endpoint.v3.ClusterLoadAssignment.Policy.weighted_priority_health>`
     values were not respected when subset load balacing was enabled. The default values of 140 and false were always used.
+- area: tracers
+  change: |
+    use unary RPC calls for OpenTelemetry trace exports, rather than client-side streaming connections
```

Comment:
@yanavlasov I think this has come up before with the original OTLP work. Do you remember?

Reply:
Indeed. PR #15149 stalled adding unary support to gRCP access logger.